src/encoded/schemas/library.json

```diff
@@ -183,41 +183,53 @@
             ],
             "format": "semi-controlled"
         },
+
         "fragmentation_method": {
             "title": "Fragmentation method",
-            "description": "A short description or reference of the nucleic acid fragmentation protocol used in library preparation, if applicable.",
-            "type": "string",
-            "enum": [
-                "chemical (DNaseI)",
-                "chemical (DpnII restriction)",
-                "chemical (generic)",
-                "chemical (HindIII restriction)",
-                "chemical (HindIII/DpnII restriction)",
-                "chemical (Illumina TruSeq)",
-                "chemical (MboI restriction)",
-                "chemical (micrococcal nuclease)",
-                "chemical (NcoI restriction)",
-                "chemical (Nextera tagmentation)",
-                "chemical (RNase III)",
-                "chemical (Tn5 transposase)",
-                "n/a",
-                "none",
-                "see document",
-                "shearing (Covaris generic)",
-                "shearing (Covaris LE Series)",
-                "shearing (Covaris S2)",
-                "shearing (generic)",
-                "sonication (Bioruptor generic)",
-                "sonication (Bioruptor Pico)",
-                "sonication (Bioruptor Plus)",
-                "sonication (Bioruptor Twin)",
-                "sonication (Branson Sonifier 250)",
-                "sonication (Branson Sonifier 450)",
-                "sonication (generic microtip)",
-                "sonication (generic)",
-                "sonication (Sonics VCX130)"
-            ]
+            "type": "array",
+            "uniqueItems": true,
+            "description": "A list of nucleic acid fragmentation methods and enzymes used in library preparation, if applicable.",
+            "items":{
+                "title": "Fragmentation method",
+                "description": "A short description or reference of the nucleic acid fragmentation protocol used in library preparation, if applicable.",
+                "type": "string",
+                "enum": [
+                    "chemical (DNaseI)",
+                    "chemical (DpnII restriction)",
+                    "chemical (generic)",
+                    "chemical (HindIII restriction)",
+                    "chemical (HindIII/DpnII restriction)",
+                    "chemical (Illumina TruSeq)",
+                    "chemical (MboI restriction)",
+                    "chemical (micrococcal nuclease)",
+                    "chemical (NcoI restriction)",
+                    "chemical (Nextera tagmentation)",
+                    "chemical (RNase III)",
+                    "chemical (Tn5 transposase)",
+                    "chemical (MseI restriction)",
+                    "chemical (CviAII restriction)",
+                    "chemical (Csp6I restriction)",
+                    "n/a",
+                    "none",
```

Comment:
Or this.

Reply:
will require data cleanup prior to the change (we have more than 200 "none")

Reply:
I think that should be another ticket